src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/SlimefunGuideImplementation.java

```diff
@@ -63,7 +64,9 @@ default void unlockItem(Player p, SlimefunItem sfitem, Consumer<Player> callback
             research.unlock(p, true, callback);
         } else {
             p.setLevel(p.getLevel() - research.getCost());
-            research.unlock(p, false, callback);
+            boolean skipLearningAnimation = SlimefunPlugin.getRegistry().isLearningAnimationDisabled()
+                    || !SlimefunGuideSettings.hasLearningAnimationEnabled(p);
```

Comment:
Using an OR connector here will render the Player setting useless if it has been disabled server-wide.
Just something to keep in mind.

Reply:
Yeah, that was the way I intended